### PR TITLE
Bedgtf

### PR DESCRIPTION
**Pull request by lldelisle**
Enhancements:
- added the possibility to merge transcripts into one single gene representation when using gtf.

Minors:
- added a warning when a boundaries_file is used.
- for the bed track, the value of display is now checked. It should be part of ['collapsed', 'triangles', 'interleaved', 'stacked', 'interlaced', 'domain'] with domain, interlaced, and interleaved all equivalent. This fix the issue #55.

We can discuss this last part, for me it was good to keep retro-compatibility but we can decide just one word and stick to it.

/!\ This might conflict with the PR #110
/!\ I used as new properties `merge transcripts` and `prefered name` but if you want I can change to `*_*`

### CODE REVIEW

File: pygenometracks/tracksClass.py [hunk 1]
```diff
@@ -369,6 +369,10 @@ def check_file_exists(track_dict, tracks_path):
                 file_field_name = key
         # for file_field_name in ['boundaries_file', 'file', 'categories_file']:
         #     if file_field_name in track_dict:
+                if file_field_name == 'boundaries_file':
```
Comment (bgruening) [inline pygenometracks/tracksClass.py]: Can you please add a comment for us, so we remove that line of code if we go towards 1.0.

File: pygenometracks/tracks/BedTrack.py [hunk 2]
```diff
@@ -118,6 +126,19 @@ def __init__(self, *args, **kwarg):
             else:
                 self.colormap = self.properties['color']
 
+        # check the display is valid:
+        if self.properties['display'] in ['interlaced', 'domain']:
```
Comment (bgruening) [inline pygenometracks/tracks/BedTrack.py]: can you define both valid/invalid ontop of the file after the imports with capital letters ... something like ... DISPLAY_BED_VALID = [...] and DISPLAY_BED_INVALID = [..] then we can reuse them and have a central place to change it....

File: pygenometracks/tracksClass.py (hunk 1 above)
Comment (lldelisle) [reply pygenometracks/tracksClass.py]: Done

File: pygenometracks/tracks/BedTrack.py (hunk 2 above)
Comment (lldelisle) [reply pygenometracks/tracks/BedTrack.py]: Done

**Comment by bgruening** [comment]
@lldelisle a final rebase is needed here, sorry.

**Comment by lldelisle** [comment]
No problem.

**Comment by LeilyR** [comment]
could you please merge the current deeptools:develop to your branch

**Comment by lldelisle** [comment]
This is done...

**Comment by lldelisle** [comment]
But it was done before @bgruening  commented that's why you do not see it after the comments...

**Comment by LeilyR** [comment]
sure

**Pull request merged**